### ISSUE TITLE
Incorrect link on docs

### ISSUE THREAD

**Issue by diegozea** [FluxML/Flux.jl#2380]
Hi!

I am trying to learn to use Flux, and I found this link error in the tutorial: https://fluxml.ai/Flux.jl/stable/tutorials/logistic_regression/#Dataset
In particular, _"(see Performance tips)"_ links to https://fluxml.ai/Flux.jl/stable/tutorials/logistic_regression/tutorials/@ref%20man-performance-tips instead of linking to https://fluxml.ai/Flux.jl/stable/performance/#[Performance-Tips]((@id-man-performance-tips))

I haven't tested, but could it be a more general problem due to a re-organization of the docs? 

Cheers,

**Comment by ToucheSir** [comment]
I don't think so, the `@ref` just needs to be to`id-man-performance-tips` instead of `man-performance-tips`. Either something changed with how Documenter.jl handles section title names, or the original page had a typo.

**Comment by diegozea** [comment]
@CarloLucibello, it looks like the issue was not solved. The link is going to https://fluxml.ai/Flux.jl/dev/tutorials/logistic_regression/tutorials/@ref%20id-man-performance-tips (404 error).

**Comment by darsnack** [comment]
Why not just do `@ref "Performance tips"` which is the Documenter default?

**Comment by ToucheSir** [comment]
I wasn't aware that syntax worked, worth a try!
